phpunit.xml.dist

```diff
@@ -19,9 +18,9 @@
             <directory>tests/TestCase/Database/</directory>
             <directory>tests/TestCase/ORM/</directory>
         </testsuite>
-        <testsuite name="http-interop-http-factory">
-            <directory>./vendor/http-interop/http-factory-tests/test</directory>
-        </testsuite>
+<!--        <testsuite name="http-interop-http-factory">-->
+<!--            <directory>./vendor/http-interop/http-factory-tests/test</directory>-->
+<!--        </testsuite>-->
```

Comment:
Should remove if this isn't needed.

Reply:
Keeping this comment for now will help us remember to add back the http-interop tests package when they eventually support phpunit 10.

Reply:
I was curious what replaced this.